src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import ProctoredExamSettings from './ProctoredExamSettings';
```

Comment:
Could you rename the component as it exists in the other file to match the name you're using to refer to it as? I for a sec thought there might be a non-proctored kind of `ExamSettings`, since the name you defined & the name you import it as don't match up.

(I understand this is really a comment on an old diff, but figure now's a good time to make this change.)